lib/src/content/components/paginating_group/navigation_toolbar_animated_builder.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+import 'wolt_modal_sheet_page_transition_state.dart';
+
+class NavigationToolbarAnimatedBuilder extends StatelessWidget {
+  final AnimationController controller;
+  final Widget child;
+  final WoltModalSheetPageTransitionState pageTransitionState;
+
+  const NavigationToolbarAnimatedBuilder({
+    required this.pageTransitionState,
+    required this.controller,
+    required this.child,
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final opacity = pageTransitionState.navigationToolbarOpacity(controller);
+    return AnimatedBuilder(
+      animation: controller,
+      builder: (BuildContext _, Widget? __) {
+        return Opacity(opacity: opacity.value, child: child);
```

Comment:
```suggestion
        return Opacity(
          opacity: opacity.value,
          child: child,
        );
```

When there are more than one parameter use a new line, Better for readibility.